packages/react/src/experimental/AiChat/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { CopilotKit } from "@copilotkit/react-core"
+import { CopilotPopup } from "@copilotkit/react-ui"
+// import "@copilotkit/react-ui/styles.css"
+
+import type { ComponentProps } from "react"
+import {
+  AssistantMessage,
+  ChatButton,
+  ChatHeader,
+  ChatTextarea,
+  ChatWindow,
+  MessagesContainer,
+  UserMessage,
+} from "./components"
+
+export type AiChatProps = ComponentProps<typeof CopilotPopup>
+
+export type AiChatProviderProps = Omit<
+  ComponentProps<typeof CopilotKit>,
+  "window"
+>
+
+export const AiChatProvider = (props: AiChatProviderProps) => {
```

Comment:
could you add the experimentalComponent wrapper function like in packages/react/src/experimental/OneDataCollection/index.tsx?
import { experimentalComponent } from "@/lib/experimental"